Examples/restcomm-olympus/app/src/main/res/layout/activity_signin.xml

```diff
@@ -10,21 +10,10 @@
               android:paddingTop="@dimen/activity_vertical_margin"
               tools:context="org.restcomm.android.olympus.SigninActivity">
 
-    <!-- Login progress -->
```

Comment:
Now that push configuration is introduced, the whole thing is a bit cluttery. I did some experiments to avoid back and forth and here's the activity_signin.xml I came up with. I just added headers and added some more spacing in the headers. Can you update the xml with this and let me know if all good? :

```
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
              xmlns:tools="http://schemas.android.com/tools"
              android:layout_width="match_parent"
              android:layout_height="match_parent"
              android:gravity="center_horizontal"
              android:orientation="vertical"
              android:paddingBottom="@dimen/activity_vertical_margin"
              android:paddingLeft="@dimen/activity_horizontal_margin"
              android:paddingRight="@dimen/activity_horizontal_margin"
              android:paddingTop="@dimen/activity_vertical_margin"
              tools:context="org.restcomm.android.olympus.SigninActivity">

    <TextView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="@string/welcome_to_olympus"
        android:id="@+id/label_call"
        android:layout_marginTop="0dp"
        android:layout_marginBottom="15dp"
        android:gravity="center_horizontal"
        android:textSize="26sp" />

    <ScrollView
        android:id="@+id/login_form"
        android:layout_width="match_parent"
        android:layout_height="match_parent">

        <LinearLayout
            android:id="@+id/email_login_form"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:orientation="vertical">

            <TextView
                android:id="@+id/sip_configuration_label"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:textStyle="bold"
                android:textColor="@color/colorAccent"
                android:text="SIP Configuration"
                android:layout_marginTop="8dp"
                android:layout_marginBottom="8dp"/>

            <android.support.design.widget.TextInputLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content">

                <EditText
                    android:id="@+id/signin_username"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:hint="@string/username"
                    android:inputType="text"
                    android:maxLines="1"
                    android:singleLine="true"/>

            </android.support.design.widget.TextInputLayout>

            <android.support.design.widget.TextInputLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content">

                <EditText
                    android:id="@+id/signin_password"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:hint="@string/password"
                    android:imeActionId="@+id/login"
                    android:imeActionLabel="@string/action_sign_in_short"
                    android:imeOptions="actionUnspecified"
                    android:inputType="textPassword"
                    android:maxLines="1"
                    android:singleLine="true"/>

            </android.support.design.widget.TextInputLayout>

            <android.support.design.widget.TextInputLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content">

                <EditText
                    android:id="@+id/signin_domain"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:hint="@string/domain"
                    android:inputType="text"
                    android:maxLines="1"
                    android:singleLine="true"/>

            </android.support.design.widget.TextInputLayout>

            <TextView
                android:id="@+id/push_configuration_label"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:text="Push Configuration"
                android:textStyle="bold"
                android:textColor="@color/colorAccent"
                android:layout_marginTop="8dp"
                android:layout_marginBottom="8dp"/>

            <android.support.v7.widget.SwitchCompat
                android:id="@+id/switch_enable_push"
                android:layout_marginStart="4dp"
                android:layout_marginTop="8dp"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:text="@string/enable_push_notifications"/>

            <LinearLayout
                android:id="@+id/ll_push_container"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:orientation="vertical">

                <android.support.design.widget.TextInputLayout
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content">

                    <EditText
                        android:id="@+id/push_account_email"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/push_account_email"
                        android:inputType="textEmailAddress"
                        android:maxLines="1"
                        android:singleLine="true"/>

                </android.support.design.widget.TextInputLayout>

                <android.support.design.widget.TextInputLayout
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content">

                    <EditText
                        android:id="@+id/push_password"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/push_account_password"
                        android:inputType="textPassword"
                        android:maxLines="1"
                        android:singleLine="true"/>

                </android.support.design.widget.TextInputLayout>

                <android.support.design.widget.TextInputLayout
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content">

                    <EditText
                        android:id="@+id/push_domain"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/push_domain"
                        android:inputType="text"
                        android:maxLines="1"
                        android:singleLine="true"/>

                </android.support.design.widget.TextInputLayout>

                <android.support.design.widget.TextInputLayout
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content">

                    <EditText
                        android:id="@+id/http_domain"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/http_domain"
                        android:inputType="text"
                        android:maxLines="1"
                        android:singleLine="true"/>

                </android.support.design.widget.TextInputLayout>

            </LinearLayout>
            <Button
                android:id="@+id/signin_button"
                style="?android:textAppearanceSmall"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_marginTop="16dp"
                android:text="@string/sign_in"
                android:textStyle="bold"
                android:textColor="@color/colorButtonWhite"
                android:backgroundTint="@color/colorPrimary" />

        </LinearLayout>
    </ScrollView>
</LinearLayout>
```